experimenter/experimenter/experiments/models.py

```diff
@@ -699,6 +699,28 @@ def can_preview_to_draft(self):
     def can_preview_to_review(self):
         return self.is_preview
 
+    def can_show_remote_settings_pending(self, reviewer):
+
+        return self.publish_status in (
+            self.PublishStatus.APPROVED,
+            self.PublishStatus.WAITING,
+        ) and self.can_review(reviewer)
+
+    @property
+    def can_show_timeout_message(self):
+        return self.changes.latest_timeout()
```

Comment:
These are the right methods to do, i think I'd preface them with `should_` like the others.